sphinx/texinputs/sphinx.sty

```diff
@@ -462,8 +480,13 @@
 \spx@tempa{spx@box@}      {box_}            {3pt}{3pt}{3pt}{3pt}
 % define legacy verbatimsep key as alias of pre_padding key
 \expandafter\let\expandafter\KV@sphinx@verbatimsep\csname KV@sphinx@pre_padding\endcsname
-% define legacy shadowsep key as alias of div.topic_padding key
-\expandafter\let\expandafter\KV@sphinx@shadowsep\csname KV@sphinx@div.topic_padding\endcsname
+% let legacy shadowsep key set all topic/contents/sidebar padding
+% keys to the common value given by user to shadosep
+\def\KV@sphinx@shadowsep #1{%
+    \@nameuse{KV@sphinx@div.topic_padding}{#1}%
```

Comment:
Ditto.

Reply:
my LaTeX varies indeed I will reduce to 2-spaces indent after committing suggested changes